### DISCUSSION TITLE
email verification & languages

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#935] [Q&A (Help)]
Following https://backpackforlaravel.com/docs/6.x/base-how-to#enable-email-verification-in-backpack-routes-1 i've enable email verification for my project that have locale= 'it' in app.php
I've seen that default /vendor/--> I've seen that language files, except the English one, don't have this block:
```
    'verify_email' => [
        'email_verification' => 'Email Verification',
        'verification_link_sent' => 'A verification link has been sent to your email address.',
        'email_verification_required' => 'Please verify your email address, by clicking on the link we\'ve sent you.',
        'resend_verification_link' => 'Resend link',
    ],
```

I've translated it for Italian:
```
    'verify_email' => [
        'email_verification' => 'Verifica Email',
        'verification_link_sent' => 'Un collegamento di verifica è stato inviato al tuo indirizzo email.',
        'email_verification_required' => 'Verifica il tuo indirizzo email facendo clic sul collegamento che ti abbiamo inviato.',
        'resend_verification_link' => 'Invia nuovamente il link di verifica',
    ],
```
Is it a lack or is it intentional?

**Comment by jcastroa87** [comment]
Hello @blondie63 

We have not yet added the translation of that section to other languages, it would be great if some native language users can support us by creating the translation or reviewing the current ones.

Cheers.

**Reply by blondie63** [reply]
You can use my IT translation

**Comment by jcastroa87** [comment]
Thanks @blondie63 i create a [PR](https://github.com/Laravel-Backpack/CRUD/pull/5658) with this translation.

Next time you are free to create a PR and take the credits for it :)

Thanks again.
